### PR TITLE
Add 404.html for general 404 errors

### PR DESCRIPTION
**Pull request by Ladsgroup**
Bug: https://phabricator.wikimedia.org/T113114

### CODE REVIEW

File: ores/wsgi/templates/404.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+a:hover { text-decoration: underline; }
+em { font-style: normal; color: #777; }
+</style>
+<div class="content" role="main">
+<a href="{{ host }}"><img src="{{ host }}/static/images/wmf.png" srcset="{{ host }}/static/images/wmf-2x.png 2x" alt="{{ alt }}" width=135 height=135></a>
+<h1>Page not found</h1>
+<p><em>{{ title }}</em></p>
+<p>We could not find the above page on our servers.</p>
+<div style="clear:both;"></div>
+<p>Alternatively, you can visit the <a href="/">Main Page</a> or read <a href="https://en.wikipedia.org/wiki/HTTP_404" title="Wikipedia: HTTP 404">more information</a> about this type of error.</p>
```
Comment (halfak) [inline ores/wsgi/templates/404.html]: Should we direct people to English Wikipedia's main page?  Seems we should try to direct them to ORES main page.

Comment (Ladsgroup) [inline ores/wsgi/templates/404.html]: I don't think we should direct people to English Wikipedia's main page. In other places it's the root path so for example https://fa.wikipedia.org/Example will direct people to Persian Wikipedia's main page. I think we should send them to ORES main page (which is the current state)

**Comment by Pix1234** [comment]
Or maybe direct to a multi-lingul 404 page?

**Comment by Ladsgroup** [comment]
@Pix1234 Nope, see the bug: https://phabricator.wikimedia.org/T113114 This is more in production cluster level.

File: ores/wsgi/templates/404.html (hunk 1 above)
Comment (halfak) [inline ores/wsgi/templates/404.html]: Oh!  i think I misread.

**Pull request merged**